### ISSUE TITLE
Mog locker doesnt update to reduced time if changed from whitegate to all areas

### ISSUE THREAD

**Issue by KnowOne134** [EdenServer/community#3505]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

Fubruhn doesn't properly update when changing accessibility. I'm uncertain if there is some dialogue missing or if the server just fails to calculate the proper time remaining and update the player. Currently the setting will change, but it appears as though it were bugged, as there is no feedback given to the player.

The time also doesn't reduce the current amount by 5/7
